.github/workflows/ci.yml

```diff
@@ -40,8 +40,8 @@ jobs:
           apt-get update
           apt-get -yqq install libpq-dev postgresql-client
       - uses: actions/checkout@v2
-      - name: Install Chromedriver
-        run: apt-get -yqq install chromium-chromedriver
+      - name: Install browser
+        run: apt-get -yqq install chromium-browser
```

Comment:
But doesn't this mean that it's an extra install step? Because you need the driver either way, right?

Reply:
Yeah, this would just dictate the **browser** we're using in the Docker container, letting the chromedriver shard pick the appropriate binary to download and test with from the chromedriver side.

At least, that's my understanding!

If we didn't run this in a Crystal docker container, the default GitHub actions container already has chromium installed. That said, then we'd have to install Crystal so it might be a wash.

Reply:
Yes, so the code will later install the driver, but I believe this is the best decision because it will more closely match how users are/should be testing their applications.

Reply:
I also realized I made this change in the actual CLI workflow file. Should still be relevant, but just FYI.